gem/oq-platform
# Hazard viewer - display multiple curves dynamically

bug: https://bugs.launchpad.net/oq-platform/+bug/1267145

This PR includes a second iteration of the hazard viewer which allows the user to select a hazard curve layer that contains multiple curves. The app will then provide the user with a list of curves in a checkbox list that can be plotted in the d3.js chart. 

To test: 
load the app
select 'hazard curves' from the tool bar ribbon
select the 'California' sample data sets from the category list
select one of the California sample data sets from the Curve Layer drop-down list
select 'add layer'
*\* Test *\* the dialog should now provide a list of curves available in the curve layer
*\* Test *\* make sure that you can select and un-select curves that are to be plotted in the chart (on map point selection (next step))
Open the Chart Dialog from the tool bar ribbon
Navigate the map to California, and select a point from the grid of curves
*\* Test *\* check that the chart dialog has a chart of selected curves plotted in it.
*\* Test *\* check that the chart has interactivity - the user can hover the mouse over each point on the chart and view the value of that point just above the chart.
